x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/footer.tsx

```diff
@@ -107,27 +110,44 @@ export const EventDetailsFooterComponent = React.memo(
     const { closeAddEventFilterModal, isAddEventFilterModalOpen, onAddEventFilterClick } =
       useEventFilterModal();
 
+    const renderFooterBody = useMemo(() => {
```

Comment:
with that amount of props I'm not sure if we get any performance improvements by using memoization here

Reply:
Alright, are you suggesting to remove it?

Reply:
I think it's okay as long as all the functions in it are memoized as well